### PR TITLE
--inspect can no longer be used as a flag through gulp

### PR DESCRIPTION
**Pull request by andrewbranch**
gulp-cli removes `--inspect` and `--inspect-brk` from `process.argv` and passes it straight to the Node process that’s running gulp, which is not the same Node process that runs mocha for us, so our test runner never sees `--inspect` as a flag. Using `-i` also wasn’t working, as that launched node with `--inspect-brk=true`, so I fixed that, and updated docs.

### CODE REVIEW

**Comment by weswigham** [comment]
...wait, how did this change? The last publish of `gulp` itself is 2 years ago, and `gulp-cli` 8 months ago. How long has this been broken for... (I mean, I guess I invoke `mocha` directly in my daily workflow, so I'd not have noticed)? o.O

**Comment by andrewbranch** [comment]
It’s been broken [for 8 months](https://github.com/gulpjs/v8flags/pull/55).

**Comment by andrewbranch** [comment]
So, I tested `--debug` and found that doesn’t get stripped, but now seeing how this works, it looks like if you were using a version of node where you could use `--debug`, that _would_ get stripped, since the recognized flags are determined dynamically. We can probably just delete that option, or alias it to `--inspect`.

**Comment by weswigham** [comment]
Maybe we should just rename it to a 3rd name that doesn't conflict? `--break`?

**Comment by andrewbranch** [comment]
`--debug` has been removed, `--break` has been added.

File: scripts/build/options.js [hunk 1]
```diff
@@ -4,12 +4,11 @@ const os = require("os");
...
 module.exports = minimist(process.argv.slice(2), {
-    boolean: ["debug", "dirty", "light", "colors", "lint", "lkg", "soft", "fix", "failed", "keepFailed", "force", "built"],
-    string: ["browser", "tests", "inspect", "host", "reporter", "stackTraceLimit", "timeout", "shards", "shardId"],
+    boolean: ["dirty", "light", "colors", "lint", "lkg", "soft", "fix", "failed", "keepFailed", "force", "built"],
+    string: ["browser", "tests", "break", "host", "reporter", "stackTraceLimit", "timeout", "shards", "shardId"],
     alias: {
         "b": "browser",
-        "d": ["debug", "debug-brk"],
-        "i": ["inspect", "inspect-brk"],
+        "i": ["inspect", "inspect-brk", "break"],
```
Comment (weswigham) [inline scripts/build/options.js:11]: should we also add `debug` here, on the offchance it both works and someone's muscle memory causes them to pass `--debug`?

Comment (andrewbranch) [reply scripts/build/options.js:11]:
```suggestion
        "i": ["inspect", "inspect-brk", "break", "debug", "debug-brk"],
```

File: Gulpfile.js [hunk 2]
```diff
@@ -453,8 +453,7 @@ task("runtests").flags = {
     "-t --tests=<regex>": "Pattern for tests to run.",
     "   --failed": "Runs tests listed in '.failed-tests'.",
     "-r --reporter=<reporter>": "The mocha reporter to use.",
-    "-d --debug": "Runs tests in debug mode (NodeJS 6 and earlier)",
-    "-i --inspect": "Runs tests in inspector mode (NodeJS 8 and later)",
+    "-i --break": "Runs tests in inspector mode (NodeJS 8 and later)",
```
Comment (DanielRosenwasser) [inline Gulpfile.js:456]: Do we really want `-i` to be the shorthand for `--break?

Comment (andrewbranch) [reply Gulpfile.js:456]: 🤷 it's backward compatible

Comment (DanielRosenwasser) [reply Gulpfile.js:456]: I'd sorta doubt anyone is taking a hard automated dependency on it, so I wouldn't worry about that too much. 😄

Either way I'll leave it up to you

Comment (andrewbranch) [reply Gulpfile.js:456]: `-b` is already taken by `--browser` despite not being documented in the Gulpfile, so I’m not sure what else to call it.

Comment (DanielRosenwasser) [reply Gulpfile.js:456]: Well...who even runs that one? Anyway, it's not a big deal.

**Pull request merged**